src/libstd/sys/unix/stdio.rs

```diff
@@ -11,11 +11,25 @@
 use io;
 use libc;
 use sys::fd::FileDesc;
+use sys::fs::{File, OpenOptions};
+use ffi::CStr;
 
 pub struct Stdin(());
 pub struct Stdout(());
 pub struct Stderr(());
 
+// FIXME: This duplicates code from process_common.rs.
```

Comment:
Ah yeah, would it be possible to extract the two to common code?

Reply:
Yeah, where should it live? `File::open_bit_bucket` maybe?